app/assets/javascripts/main.js

```diff
@@ -0,0 +1,191 @@
+/* eslint-disable no-unused-expressions */
+/* global $, Locations */
+
+;(function () {
```

Comment:
It's a safety fallback for when we concatenate multiple JS files. It's fairly common practice I think.